test/data/mock.php

```diff
@@ -229,6 +229,17 @@ protected function errorWithScript( $req ) {
 		}
 	}
 
+	protected function cors($req) {
+		if (isset($req->headers['ORIGIN'])) {
+			$origin = $req->headers['ORIGIN'];
+			header("Access-Control-Allow-Methods: GET");
+			header("Access-Control-Allow-Origin: " . $origin);
+			corsCallback(true);
```

Comment:
There's a bug here, this will not actually do anything. It should be:
```suggestion
			echo 'corsCallback( true )';
```

Reply:
Hmm, actually, this mistake was why the PHP tests completely failed for me... With these changes they work even in Safari.

Taking that & what I wrote in:
https://github.com/jquery/jquery/pull/4563#issuecomment-575833262
into account, perhaps the current check is fine, we should just add a comment above the `localhost` to `127.0.0.1` replacement saying that full cross-origin verification is only done when tests run from `localhost`, e.g. when run via Karma (as we can depend on `localhost` resolving to `127.0.0.1`) and that in other cases we only do a more limited check that the Origin header is sent.